protocol/v2/qbft/controller/controller.go

```diff
@@ -71,6 +84,7 @@ func (c *Controller) StartNewInstance(logger *zap.Logger, height specqbft.Height
 	newInstance := c.addAndStoreNewInstance()
 	newInstance.Start(logger, value, height)
 	c.forceStopAllInstanceExceptCurrent()
+	metricsDutiesCreated.WithLabelValues().Inc()
```

Comment:
not sure if this is just a test or not, but we already have similar metrics inside each Runner (see `r.metrics.Consensus()` for starting instance and `r.metrics.EndConsensus()` for decided)

i'd check if it's possible to work with the existing implementation of Runner metrics instead of adding more metric calls